helm/superset/README.md

```diff
@@ -132,6 +132,8 @@ On helm this can be set on `extraSecretEnv.SUPERSET_SECRET_KEY` or `configOverri
 | supersetCeleryBeat.enabled | bool | `false` | This is only required if you intend to use alerts and reports |
 | supersetCeleryBeat.extraContainers | list | `[]` | Launch additional containers into supersetCeleryBeat pods |
 | supersetCeleryBeat.forceReload | bool | `false` | If true, forces deployment to reload on each upgrade |
+| supersetCeleryBeat.image.repository | string | `nil` |  |
+| supersetCeleryBeat.image.tag | string | `nil` |  |
```

Comment:
Can we also add `pullPolicy` so that it has the same fields as `.image`? https://github.com/apache/superset/blob/f2e020e39837382a339c5f008abec53f584761df/helm/superset/values.yaml#L186-L189